homeassistant/components/mqtt/fan.py

```diff
@@ -190,7 +180,7 @@ def valid_preset_mode_configuration(config):
 DISCOVERY_SCHEMA = vol.All(
     # CONF_SPEED_COMMAND_TOPIC, CONF_SPEED_LIST, CONF_SPEED_STATE_TOPIC, CONF_SPEED_VALUE_TEMPLATE and
     # Speeds SPEED_LOW, SPEED_MEDIUM, SPEED_HIGH SPEED_OFF,
-    # are deprecated, support will be removed with release 2021.9
+    # are no longer supported, support was removed in release 2021.9
```

Comment:
Will be probably removed with 2022.2

Reply:
We removed support with https://github.com/home-assistant/core/pull/58992, which was in 2021.12 so I guess that's what it should say?

Reply:
Yes, of course